### DISCUSSION TITLE
Error with AveYo script when trying to install WIN11 on unsupported PC

### DISCUSSION THREAD

**Discussion by poozu** [AveYo/MediaCreationTool.bat#233] [Q&A] [answered]
Hi,

I am trying to install WIN11 on unsuported HW (CPU and TPM issue).
When running the AveYo script Skip_TPM_Check_on_Dynamic_Update.cmd found at https://github.com/AveYo/MediaCreationTool.bat/tree/main/bypass11

I get an ACCESS DENIED error in the batch window. The script is run as an admin.

Thanks for suggestions.

**Comment by lewygrega** [comment]
I have the same problem on a new PC with different IP address on the same network. It seems I am on M$ black list AND  that changing IP with my ISP did not suffice. Any ideas? Thank You  in advance, sincerely, Grega

**Comment by jeffclayton** [comment]
A friend of mine was having a problem as well. I had him download the iso from MS first for good measure, and then use regedit to enter Microsoft's own code. No other method seemed to work. I was not there to observe, but maybe entering the key first may help you?

Location:

HKEY_LOCAL_MACHINE\SYSTEM\Setup\MoSetup

Create a DWORD (32-bit) Value called AllowUpgradesWithUnsupportedTPMOrCPU

Set AllowUpgradesWithUnsupportedTPMOrCPU to 1

Rebooted machine and it allowed him to use the installation media after that point.

**Comment by lewygrega** [comment]
As I understand it, M$ is blocking repositores of people that was reported
to use this script.

**Reply by jeffclayton** [reply]
I had thought that might be what you were getting at and hoped that wasn't the case. My friend had the script (I sent him the link) but he did also have that reg patch and the iso. He might not have ended up using the script at all. In experience with installations, some non-script workarounds have been to install a version of windows (before activation) with the net disconnected. If it is possible to do it without the script -- or even without an internet connection, then a different multi-stage new script version might be workable?? I can't imagine this to be impossible. He did have a valid version of Windows 10 on that old machine -- if that isn't a non-requirement, so it was supposedly completely legitimate by the rules as I understand them to be.

**Comment by poozu** [comment, marked as answer]
What worked for me was to run the AveYo fullscript (mediacreationtool.bat) and install 22H1. I then downloaded the ISO for 22h2 and deleted compatappraiserresources.dll in the sources folder. That did install 22H2 without "access denied"

**Reply by jeffclayton** [reply]
Great tip!!!

**Reply by jeffclayton** [reply]
It might then be possible to adjust the script to at least delete that file but also do a version detection check.
